pkg/gatewayserver/io/io.go

```diff
@@ -265,7 +269,22 @@ func (c *Connection) ScheduleDown(path *ttnpb.DownlinkPath, msg *ttnpb.DownlinkM
 	if err != nil {
 		return 0, err
 	}
-	phy, err := band.GetByID(c.fp.BandID)
+	var fp *frequencyplans.FrequencyPlan
+	for _, gtwFP := range c.fps {
+		reqFP, err := c.fpStore.GetByID(request.FrequencyPlanID)
```

Comment:
We should not be getting the frequency plans from the store in the hot path; we have the frequency plans already loaded in the connection's gateway right?